srcpkgs/mattermost-desktop/template

```diff
@@ -2,18 +2,48 @@
 pkgname=mattermost-desktop
 version=4.5.2
 revision=1
-archs="x86_64 i686"
 wrksrc="desktop-${version}"
+archs="x86_64* i686*"
 # electron-builder needs GNU tar
-hostmakedepends="git nodejs-lts-10 python libicns GraphicsMagick xz tar"
-makedepends="libnotify-devel"
+hostmakedepends="git jq nodejs-lts-10 python libicns GraphicsMagick xz tar"
+makedepends="libnotify-devel electron7"
+depends="electron7"
 short_desc="Team messaging app, an open source Slack alternative"
 maintainer="Artem Zhurikhin <ashpool@xecut.net>"
 license="Apache-2.0"
 homepage="https://mattermost.com/"
 distfiles="https://github.com/mattermost/desktop/archive/v${version}.tar.gz"
 checksum=17e32346592d175f091a4f12068e0cfc5cfd20ed7894f88bb5b54b2eb6eac74a
 
+if [ "$XBPS_LIBC" = "musl" ]; then
+	hostmakedepends+=" gcompat" #XXX: ugly af
+	# Npm just fetches a 7za binary that is lined against glibc
+	# refuses to use the system one
+fi
+
+do_configure() {
+	if [ "${XBPS_TARGET_MACHINE#-musl}" = x86_64 ];then
+		sed -i 's/--ia32//g' package.json
+	elif [ "${XBPS_TARGET_MACHINE#-musl}" = i686 ];then
+		sed -i 's/--x64//g' package.json
+	fi
```

Comment:
Isn't using `case` a bit easier here? Would help if we get cross electron at some point too.